imagine-dragons-web/songs.html

```diff
@@ -5,50 +5,57 @@
+                <a id="col4" href="https://www.youtube.com/watch?v=7wtfhZwyrcc" target="_blank">Click here to listen</a>
             </section>
-            <section>
-                <h3>Thunder</h3>
-                <p>The song was released by Interscope Records and Kidinakorner on April 27, 2017, as the second single from their third studio album, Evolve.
+            <section class="song">
+                <h3 id="col1">Thunder</h3>
+                <img id="col2" src="resources/images/thunder-song.jpg">
+                <p id="col3">The song was released by Interscope Records and Kidinakorner on April 27, 2017, as the second single from their third studio album, Evolve.
                    Written by Dan Reynolds, Wayne Sermon, Ben McKee, Daniel Platzman, and its producers Alexander Grant and Jayson DeZuzio, "Thunder" peaked at number four on the US Billboard Hot 100.
                    The song became the band's fourth single in that country to reach the top ten after "Radioactive", "Demons" and "Believer".
                    The song was nominated for a Grammy Award for Best Pop Duo/Group Performance. </p>
-                <a href="https://www.youtube.com/watch?v=fKopy74weus" target="_blank">Click here to listen</a>
+                <a id="col4" href="https://www.youtube.com/watch?v=fKopy74weus" target="_blank">Click here to listen</a>
             </section>
-            <section>
-                <h3>Radioactive</h3>
-                <p>The song received positive reviews from critics, who praised the production, lyrics, and vocals, calling it a highlight on the album.
+            <section class="song">
+                <h3 id="col1">Radioactive</h3>
+                <img id="col2" src="resources/images/radioactive-song.jpg">
+                <p id="col3">The song received positive reviews from critics, who praised the production, lyrics, and vocals, calling it a highlight on the album.
                    Due to heavy rotation on various commercials and trailers, the song became a sleeper hit, peaking at number three on the US Billboard Hot 100 chart and becoming the band's first top 10 single as well as being the third best selling song in that country in 2013.
                    It also broke the record for slowest ascension to the top 5 in chart history and currently holds the record for most weeks spent on the Billboard Hot 100 at 87 weeks</p>
-                <a href="https://www.youtube.com/watch?v=ktvTqknDobU" target="_blank">Click here to listen</a>
+                <a id="col4" href="https://www.youtube.com/watch?v=ktvTqknDobU" target="_blank">Click here to listen</a>
             </section>
-            <section>
-                <h3>On Top Of The World</h3>
-                <p>On 13 November 2013 the band released an official music video for the song.
+            <section class="song">
+                <h3 id="col1">On Top Of The World</h3>
```

Comment:
what is the difference between an `id` and `class`? which you should be using here?

Reply:
id should be used only for a single element (identifier - meaning it is special for this element), therefore when I use the same id for multiple element it was wrong and I should have use class selector instead